experimental/transliterator_parser/src/lib.rs

```diff
@@ -300,14 +301,18 @@ mod tests {
                 rule_group_list: VarZeroVec::from(&expected_rule_group_list),
                 variable_table: expected_var_table,
                 visibility: true,
-                dependencies: VarZeroVec::from(&[
```

Comment:
This field got accidentally readded for the reverse case during the merge (line 409)